--- a/Makefile.in-SAVE	2014-02-23 20:07:52.000000000 -0800
+++ b/Makefile.in	2014-08-12 13:45:25.324317818 -0700
@@ -16,7 +16,7 @@
 QUOTA_LIB_SUBDIR= lib/quota
 
 LIB_SUBDIRS=lib/et lib/ss lib/e2p $(UUID_LIB_SUBDIR) lib/ext2fs $(BLKID_LIB_SUBDIR) $(QUOTA_LIB_SUBDIR) intl
-PROG_SUBDIRS=e2fsck $(DEBUGFS_DIR) misc $(RESIZE_DIR) tests/progs po
+PROG_SUBDIRS=e2fsck
 SUBDIRS=util $(LIB_SUBDIRS) $(PROG_SUBDIRS) tests
 
 SUBS= util/subst.conf lib/config.h lib/dirpaths.h \
